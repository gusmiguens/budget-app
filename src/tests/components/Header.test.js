const React = require('react')
const { shallow } = require('enzyme')
const Header = require('../../components/Header')

describe('<Header />', () => {
    it('Should render Header correctly', () => {
        const wrapper = shallow(<Header />)
        expect(wrapper).toMatchSnapshot()
    })
})