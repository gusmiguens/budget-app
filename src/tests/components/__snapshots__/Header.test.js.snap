// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Should render Header correctly 1`] = `
<header>
  <h1>
    Budget
  </h1>
  <NavLink
    activeClassName="is-active"
    to="/dashboard"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/create"
  >
    Create Page
  </NavLink>
  <button
    onClick={[Function]}
  >
    Logout
  </button>
</header>
`;
